src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -265,7 +262,7 @@ void manualRegistrationToolBox::stopManualRegistration()
 
 void manualRegistrationToolBox::synchroniseMovingFuseView()
 {
-    if(this->parentToolBox()) //no need in Reg. workspace
+    if(selectorToolBox()) //no need in Reg. workspace
```

Comment:
Actually there's no selectorToolBox if this toolbox is called outside of Registration Workspace, e.g in pipelines.
So we need these lines to have the tbxes compatible with pipelines.